script.gs

```diff
@@ -120,35 +114,65 @@ var cardStartHeaders = {
   'Are you looking for emotional or spiritual support?': 'emotional',
   'Do you need resource support, or support with anything not on this form already?':
     'resources',
+  'Anything about yourself you’d like to share (zero expectation if you’d rather not)':
```

Comment:
This is in the form in the first section already, under `Personal Info` (line 238)

Reply:
I actually use that as the marker for when to end the resources support. You can see the other section is disabled. I couldn’t really come up with a better way to mark the “end” of the last card.

Reply:
Ohhh gotcha – that makes sense now that you explain it that way! I don't mind it being there then. The only other way that comes to mind as a way to "end" would be to look for `i = responseLength - 4`, which is when we know we're in the last section, but that seems no better than what you have. So this is good to go IMHO.

Reply:
I think this way at least there’s the possibility that if the order of the question groups or number of questions change in any group the parser will still work without changes.